config/secret_settings.yml.example

```diff
@@ -46,9 +46,16 @@ exception:
   sender: '"Sender" <sender@example.org>'
   recipients: 'recipients@example.org'
 
+mail_recipients:
+  salesforce: 'recipients@example.org'
+
 valid_iframe_origins:
   - http://localhost
   - https://cnx.org
   - https://openstax.org
 
 disable_corner_console: false
+
+salesforce:
+  consumer_key: ~
+  consumer_secret: ~
```

Comment:
Any changes to this file will conflict with the Rails 4 PR because this got renamed to `secrets.yml.example`. Also don't forget to add these to the deployment script.